components/automate-deployment/pkg/server/server.go

```diff
@@ -781,7 +781,7 @@ func (s *server) Status(ctx context.Context, d *api.StatusRequest) (*api.StatusR
 	nonSkippedServices = append(nonSkippedServices, deploymentServiceName)
 	for _, svcString := range s.deployment.ServiceNames() {
 		if svc, found := s.deployment.ServiceByName(svcString); found {
-			if svc.DeploymentState != deployment.Skip {
```

Comment:
Excluding `deployment.Skip` means that you include `Running`, `Removed`, and `Installed` states. It's not clear why the code was written to just exclude `Skip` here, but it seems to me that `Running` services are the only ones that should be expected to have a passing health check (?)

Reply:
The change you have makes sense to me as well. 👍